src/test/java/uk/gov/hmcts/darts/audio/component/impl/OutboundFileProcessorImplTest.java

```diff
@@ -390,6 +394,66 @@ void processAudioForDownloadShouldReturnThreeSessionsWithDifferentNumbersOfAudio
         verify(audioOperationService, times(3)).trim(any(), any(), any(), any());
     }
 
+    @Test
+    void processAudioForDownloadShouldReturnTwoSessionsWhenProvidedAudiosWithVaryingStartEndTimes()
+        throws ExecutionException, InterruptedException, IOException {
+
+        var firstTrimmedAudioFileInfo = new AudioFileInfo(TIME_10_01.toInstant(),
+                                                          TIME_11_00.toInstant(),
+                                                          1,null);
+        var secondTrimmedAudioFileInfo = new AudioFileInfo(TIME_11_01.toInstant(),
+                                                           TIME_11_59.toInstant(),
+                                                           1,null);
+
+        AudioFileInfo mergedAudioFile1 = new AudioFileInfo(TIME_10_00.toInstant(),
+                                                           TIME_11_00.toInstant(),
+                                                           1,null);
+
+        AudioFileInfo mergedAudioFile2 = new AudioFileInfo(TIME_11_00.toInstant(),
+                                                           TIME_12_00.toInstant(),
+                                                           1,null);
+        var reEncodedAudioFileInfo1 = firstTrimmedAudioFileInfo;
+        var reEncodedAudioFileInfo2 = secondTrimmedAudioFileInfo;
+        when(audioOperationService.reEncode(any(), any()))
+            .thenReturn(reEncodedAudioFileInfo1).thenReturn(reEncodedAudioFileInfo2);
+
+        when(audioOperationService.merge(any(), any()))
+            .thenReturn(mergedAudioFile1).thenReturn(mergedAudioFile2);
+
+        when(audioOperationService.trim(any(), any(), any(), any()))
+            .thenReturn(firstTrimmedAudioFileInfo)
+            .thenReturn(secondTrimmedAudioFileInfo);
+
+        var mediaEntity1 = createMediaEntity(
+            TIME_10_00,
+            TIME_11_00,
+            1
+        );
+        var mediaEntity2 = createMediaEntity(
+            TIME_11_01,
+            TIME_12_00,
+            2
```

Comment:
I guess the channel here should just be 1, if there is a single device.
It doesn't seem to match up with the channel in `mergedAudioFile2`?

Reply:
error in the test code - they should be both on the same channel so that concatenation occurs